accern/default_client.py

```diff
@@ -230,3 +155,21 @@ def handle_request_error(e):
                 err += " with no error message"
         msg = textwrap.fill(msg) + "\n\n(Network error: %s)" % (err,)
         raise error.APIConnectionError(msg)
+
+def quant_filter(json_data,filters):
+    signals=json_data['signals']
+    quant_variables=['event_relevance','entity_relevance','story_group_exposure','entity_sentiment','event_sentiment','story_group_sentiment_avg','story_group_sentiment_stdev','story_sentiment','story_group_count','story_group_traffic_sum','story_traffic','entity_exchange','event_source_timeliness_score','event_impact_pct_change_avg','event_impact_pct_change_stdev','event_impact_pos','event_impact_neg','event_impact_gt_mu_add_sigma','event_impact_lt_mu_sub_sigma','event_impact_gt_mu_pos_add_sigma_pos','event_impact_lt_mu_neg_sub_sigma_neg','event_impact_gt_mu_pos_add_2sigma_pos','event_impact_lt_mu_neg_sub_2sigma_neg','event_impact_gt_1pct_pos','event_impact_lt_1pct_neg']
```

Comment:
single value each line.

move global variables out of the function. put on the top of the code